webtweets/index.html

```diff
@@ -15,52 +15,63 @@
         <script src="js/loklak-fetcher.js"></script>
     </head>
     <body onload="datafetcher()">
-		<div class="nav-container"></div>
-		<div class="main-container">
-			<section class="subscribe-1">
+        <div class="nav-container"></div>
+        <div class="main-container">
+            <section class="subscribe-1">
 
-				<div class="container">
-					<div class="row">
-						<div class="col-sm-12 text-center">
-							<h1 class="large-h1 text-white">Twitter Updates</h1>
-						</div>
-					</div>
-					<div class="row">
-						<div class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 text-center">
-							<div class="twitter-feed">
-								<i class="icon social_twitter text-white"></i>
-                <!-- Change the data-count to desired number of circulating tweets and data-query to the search terms you desire. -->
-								<div class="tweets-feed" id="tweets" data-count=50 data-query="fossasia #gci">
-									<div class="arrow-up"></div>
-									<p id="tweet" class="tweet">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-sm-12 text-center">
+                            <h1 class="large-h1 text-white">Twitter Updates</h1>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 text-center">
+                            <div class="twitter-feed">
+                                <i class="icon social_twitter text-white"></i>
+                <!-- Change the data-count to desired number of circulating tweets and data-query 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/webtweets/index.html
+++ b/webtweets/index.html
@@ -29,7 +29,7 @@
                         <div class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 text-center">
                             <div class="twitter-feed">
                                 <i class="icon social_twitter text-white"></i>
-                <!-- Change the data-count to desired number of circulating tweets and data-query 
+                <!-- Change the data-count to desired number of circulating tweets and data-query
                      to the search terms you desire. -->
                                 <div class="tweets-feed" id="tweets" data-count=50 data-query="fossasia #gci">
                                     <div class="arrow-up"></div>
```